shipyard-controller/handler/sequencedispatcher.go

```diff
@@ -147,6 +146,55 @@ func (sd *SequenceDispatcher) dispatchSequences() {
 	}
 }
 
+func (sd *SequenceDispatcher) isSequenceBlocked(queueItem models.QueueItem) (bool, error) {
+	// searching for running sequences
+	startedSequenceExecutions, err := sd.sequenceExecutionRepo.Get(models.SequenceExecutionFilter{
+		Scope: models.EventScope{
+			EventData: keptnv2.EventData{
+				Project: queueItem.Scope.Project,
+				Stage:   queueItem.Scope.Stage,
+				Service: queueItem.Scope.Service,
+			},
+		},
+		Status: []string{apimodels.SequenceStartedState},
+	})
+	if err != nil {
+		return true, err
+	}
+
+	if len(startedSequenceExecutions) > 0 {
```

Comment:
can we also add a log here to print the keptnContext ID of the currently started sequence? Retrieving the ID of the currently running sequence has often been complicated, especially in support cases where we don't have access to the DB, so this would be really helpful here :)

Reply:
sure no problem :)